.circleci/config.yml

```diff
@@ -10,6 +10,15 @@ references:
         user: builder
     working_directory: ~/epoch
 
+  merge_checkout: &merge_checkout
+    run:
+      name: Checkout merged master
+      command: |
+        if [ -n "$CIRCLE_PR_NUMBER" ]; then
+          git fetch origin +refs/pull/${CIRCLE_PR_NUMBER}/merge
+          git checkout -qf FETCH_HEAD
```

Comment:
Could you echo here the resulting commit that is being tested? As of course the job will link to the branch commit rather than the merged commit.

Reply:
yep good idea, see 10ff7b4